db/migrate/20240227113858_create_villas.rb

```diff
@@ -0,0 +1,15 @@
+class CreateVillas < ActiveRecord::Migration[7.1]
+  def change
+    create_table :villas do |t|
+      t.string :name
+      t.string :address
+      t.string :movie_genre
+      t.string :description
```

Comment:
I wonder if text wouldn't be more appropriate for a description

Reply:
done !